enterprise/server/remote_execution/operation/operation.go

```diff
@@ -68,6 +68,8 @@ func (p *Publisher) Send(op *longrunning.Operation) error {
 
 // Ping re-publishes the current execution progress state.
 func (p *Publisher) Ping() error {
+	time.Sleep(750 * time.Millisecond)
```

Comment:
did you mean to include this change in this pr?

Reply:
Woops, removed